clang/test/Driver/openmp-offload-gpu.c

```diff
@@ -101,17 +101,6 @@
 
 /// ###########################################################################
 
-/// Check that the warning is thrown when the libomptarget bitcode library is not found.
-/// Libomptarget requires sm_52 or newer so an sm_52 bitcode library should never exist.
-// RUN:   not %clang -### -fopenmp=libomp -fopenmp-targets=nvptx64-nvidia-cuda \
-// RUN:   -Xopenmp-target -march=sm_52 --cuda-path=%S/Inputs/CUDA_102/usr/local/cuda \
-// RUN:   -fopenmp-relocatable-target -save-temps %s 2>&1 \
-// RUN:   | FileCheck -check-prefix=CHK-BCLIB-WARN %s
-
-// CHK-BCLIB-WARN: no library 'libomptarget-nvptx-sm_52.bc' found in the default clang lib directory or in LIBRARY_PATH; use '--libomptarget-nvptx-bc-path' to specify nvptx bitcode library
```

Comment:
When removing this test you suddenly lose code coverage for the err_drv_omp_offload_target_missingbcruntime diagnostic. That seems a bit unfortunate.

Reply:
Ok. Just wanted to raise the concern. Anyway this patch would help, to avoid failures in our downstream buildbots (and I wouldn't need to workaround those problems downstream).
So LGTM!